docs/latest/user/deployment-mode.md

```diff
@@ -0,0 +1,393 @@
+### One GatewayClass per Envoy Gateway
+
+* Envoy Gateway can accept a single [GatewayClass](https://gateway-api.sigs.k8s.io/api-types/gatewayclass/)
+resource. If you've instantiated multiple GatewayClasses, we recommend running multiple Envoy Gateway controllers
+in different namespaces, linking a GatewayClass to each of them. 
+* Support for accepting multiple GatewayClass is being tracked [here](https://github.com/envoyproxy/gateway/issues/1231).
+
+### Deployment Mode
```

Comment:
Should `Deployment Mode` be the first line, as the post title?

Reply:
sure, updated